### PR TITLE
Fix #21: Prevent the tests from being installed with the wheel file

### LINKED ISSUE #21: Installing django-ratelimit-backend includes the tests
Since 1edd1667bb840763f98430b478142de44acfb7e8, `pip install django-ratelimit-backend==1.0` installs the package's tests in `site-packages/tests` which isn't desirable.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: ratelimitbackend/admin.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-from django.contrib.admin import *  # noqa
```
Comment (brutasse) [inline ratelimitbackend/admin.py]: The idea behind `import *` is to be able to use `ratelimitbackend.admin` wherever `django.contrib.admin` is used. E.g. if you do `from ratelimitbackend import admin`, you can still write `admin.ModelAdmin` without having to import from the Django admin under another name.

Comment (timgraham) [inline ratelimitbackend/admin.py]: I don't think such transitive imports are a best practice, but if you insist, I'll revert this and try to hide the flake8 error instead.

Comment (timgraham) [inline ratelimitbackend/admin.py]: I reverted this.

**Comment by brutasse** [comment]
Thanks!

**Pull request merged**